docs/modules/blocks/enderchests.mdx

```diff
@@ -85,7 +84,7 @@ Additionally, players cannot place Ender chests under any circumstances.
           Region in which a player's Ender chest content will be dropped off.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Region</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```

Comment:
Needs to be "Randomize-able" Region, not all bounded regions will work.
As an example, a sphere region (which is bounded) won't work.